### PR TITLE
Add the manifest to build dev-libs/evdev*.

### PR DESCRIPTION
**Pull request by Arignir**
```
[!]      Some built files haven't been moved to any package:
[!]          usr/lib64/libevdev.la
[+]      Wrapping dev-libs/evdev#1.8.0
[+]          name: evdev
[+]          category: dev-libs
[+]          version: 1.8.0
[+]          description: A handler library for evdev events and devices.
[+]          tags: evdev, device
[+]          maintainer: grange_c@raven-os.org
[+]          licenses: custom
[+]          upstream_url: https://www.freedesktop.org/wiki/Software/libevdev/
[+]          kind: effective
[+]          wrap_date: 2019-11-18T16:06:34Z
[+]          dependencies:
[+]              sys-libs/glibc
[+]         
[+]          Files added:
[+]              ./usr/lib64/libevdev.so.2 -> libevdev.so.2.3.0
[+]              ./usr/lib64/libevdev.so.2.3.0
[+]              ./usr/lib64/pkgconfig/libevdev.pc
[+]              ./usr/bin/touchpad-edge-detector
[+]              ./usr/bin/mouse-dpi-tool
[+]              ./usr/bin/libevdev-tweak-device
[+]          (That's 6 files.)
[+]          Creating data.tar.gz
[+]          Creating manifest.toml
[+]          Creating evdev-1.8.0.nest
[+]      Wrapping dev-libs/evdev-dev#1.8.0
[+]          name: evdev-dev
[+]          category: dev-libs
[+]          version: 1.8.0
[+]          description: Headers and manuals to compile or write a software using the dev-libs/evdev package.
[+]          tags: evdev, device
[+]          maintainer: grange_c@raven-os.org
[+]          licenses: custom
[+]          upstream_url: https://www.freedesktop.org/wiki/Software/libevdev/
[+]          kind: effective
[+]          wrap_date: 2019-11-18T16:06:34Z
[+]          dependencies:
[+]              dev-libs/evdev#=1.8.0
[+]         
[+]          Files added:
[+]              ./usr/share/man/man3/libevdev.3
[+]              ./usr/lib64/libevdev.a
[+]              ./usr/lib64/libevdev.so -> libevdev.so.2.3.0
[+]              ./usr/include/libevdev-1.0/libevdev/libevdev.h
[+]              ./usr/include/libevdev-1.0/libevdev/libevdev-uinput.h
[+]          (That's 5 files.)
[+]          Creating data.tar.gz
[+]          Creating manifest.toml
[+]          Creating evdev-dev-1.8.0.nest
[+]      Wrapping dev-libs/evdev-doc#1.8.0
[!]      The package is empty -- Skipping
[+]  Done!
```

### CODE REVIEW

File: kernel/linux/config-5.2.17 [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 
 #
-# Compiler: gcc (Gentoo 8.3.0-r1 p1.1) 8.3.0
+# Compiler: gcc (Gentoo 8.3.0-r2 p2) 8.3.0
```
Comment (doom) [inline kernel/linux/config-5.2.17:7]: Why is this file changed ?

Comment (Arignir) [reply kernel/linux/config-5.2.17]: I edited the configuration on my host computer. This file is auto-generated and this is the compiler's version, which has changed (on my host system) since the last update.

Afaik it's no big deal, it's useful for kernel developers to know the compiler's version when the configuration is copy pasted.

Comment (Arignir) [reply kernel/linux/config-5.2.17:7]: Because evdev requires some kernel features to work properly.

**Review by doom — APPROVED** [review]
